operator-framework/src/main/java/io/javaoperatorsdk/operator/CustomResourceToCustomResourceDoneableProvider.java

```diff
@@ -0,0 +1,58 @@
+package io.javaoperatorsdk.operator;
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.client.CustomResourceDoneable;
+import io.javaoperatorsdk.operator.api.ResourceController;
+import org.apache.commons.lang3.ClassUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+class CustomResourceToCustomResourceDoneableProvider {
```

Comment:
Can't this and `ClassMappingProvider` somehow be made generic so that they share code?

Reply:
Actually, this class was supposed to be replaced by ClassMappingProvider. 
I think I just missed to remove it, will check.

Reply:
The class was indeed unused, removed it.